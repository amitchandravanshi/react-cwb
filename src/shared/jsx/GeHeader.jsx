import React from "react";
import Button from "@material-ui/core/Button";
import ShareWithOthers from "../../components/Popups/ShareWithOthers";
import "../scss/header.css";

const GeHeader = () => (
  <div>
    <header className="App-header">
      <div className="row">
        <div className="col-md-4">
          <input
            type="text"
            className="form-control headerTxt"
            placeholder="CAP-IDEAS-4-roles"
          />
          <Button
            variant="contained"
            color="primary"
            classes={{
              root: "classes-state-root"
            }}
          >
            Save
          </Button>
        </div>
        <div className="col-md-8">
          <div className="lbl-comments">
            <label>Show Resolved Comments</label>
            <ShareWithOthers />
            <span className="badge badge-light">GP</span>
          </div>
        </div>
      </div>
    </header>
  </div>
);

export default GeHeader;