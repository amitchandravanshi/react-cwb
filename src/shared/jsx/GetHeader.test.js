import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import GeHeader from "./GeHeader";
const props = {
  styles: {}
};
describe("<GeHeader> component ", () => {
  it("shallow copy component", () => {
    const wrapper = shallow(<GeHeader {...props} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
