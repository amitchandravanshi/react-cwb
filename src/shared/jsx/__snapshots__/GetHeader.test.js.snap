// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GeHeader> component  shallow copy component 1`] = `
<div>
  <header
    className="App-header"
  >
    <div
      className="row"
    >
      <div
        className="col-md-4"
      >
        <input
          className="form-control headerTxt"
          placeholder="CAP-IDEAS-4-roles"
          type="text"
        />
        <WithStyles(Button)
          classes={
            Object {
              "root": "classes-state-root",
            }
          }
          color="primary"
          variant="contained"
        >
          Save
        </WithStyles(Button)>
      </div>
      <div
        className="col-md-8"
      >
        <div
          className="lbl-comments"
        >
          <label>
            Show Resolved Comments
          </label>
          <WithStyles(ShareWithOthers) />
          <span
            className="badge badge-light"
          >
            GP
          </span>
        </div>
      </div>
    </div>
  </header>
</div>
`;
